hawc/apps/lit/templates/lit/fragments/workflow_row.html

```diff
@@ -89,6 +89,6 @@ <h4 class="mb-3">{{object.title}}&nbsp;<i class="fa fa-spinner fa-spin htmx-indi
     </div>
   </div>
   <div class="mt-2 ml-1 d-flex justify-content-center text-center">
-    <i class="m-0 text-muted" style="max-width: 800px;">{{object.description}}</i>
+    <i class="m-0 text-muted" style="max-width: 75%;">{{object.description}}</i>
```

Comment:
does this look weird on mobile or small widths screens?

Reply:
The whole page looks a bit off on smaller screens--this specific change isn't causing further problems. 
But overall the page looks good if the phone is rotated horizontally!